### PR TITLE
Handle dictionaries in zstd

### PR DESCRIPTION
**Pull request by kampersanda**
This PR modifies all CLI tools to handle zstd compressed dictionaries.

### Note 1

Although `ruzstd` was used in `dictgen`, this modification unifies the dependency to `zstd` throughout this repository because confusion can arise when different libraries with similar functions are used. Pure Rust would be preferable, but the advantages of unifying into one dependency would be greater.

### Note 2

This PR adds shell hints to code blocks to highlight comment-outs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,24 +43,26 @@ First of all, install `rustc` and `cargo` following the [official instructions](
...
+several precompiled dictionaries from different resources.
+
 Here, consider to use [mecab-ipadic v2.7.0](https://taku910.github.io/mecab/).
 
-```
-$ wget https://github.com/daac-tools/vibrato/releases/download/v0.3.1/ipadic-mecab-2_7_0.tar.gz
+```shell
+# Specify an appropriate release tag to VERSION (e.g., v0.4.0).
+$ wget https://github.com/daac-tools/vibrato/releases/download/VERSION/ipadic-mecab-2_7_0.tar.gz
 $ tar -xzf ipadic-mecab-2_7_0.tar.gz
```
Comment (vbkaisetsu) [inline README.md:54]: Could you use `.tar.xz`?

File: README.md [hunk 2]
```diff
@@ -95,7 +97,7 @@ but with the default settings it can produce different tokens from MeCab.
 
 For example, MeCab ignores spaces (more precisely, `SPACE` defined in `char.def`) in tokenization.
 
-```
+```shell
```
Comment (vbkaisetsu) [inline README.md:100]: This block contains the result of the command, and that result is highlighted incorrectly.
Blocks should be left as is.

File: README.md (hunk 1 above)
Comment (kampersanda) [reply README.md]: Sure. https://github.com/daac-tools/vibrato/pull/112/commits/1dba2931858257e83e322bdedfe0c1f636d726c4

File: README.md (hunk 2 above)
Comment (kampersanda) [reply README.md]: I reverted it for code blocks containing output results. https://github.com/daac-tools/vibrato/pull/112/commits/8cdc1ec4d8d3720f3ade4b9ae5b2dc12b30eb41d

File: README.md [hunk 3]
```diff
@@ -43,24 +43,26 @@ First of all, install `rustc` and `cargo` following the [official instructions](
...
+
 Here, consider to use [mecab-ipadic v2.7.0](https://taku910.github.io/mecab/).
 
-```
-$ wget https://github.com/daac-tools/vibrato/releases/download/v0.3.1/ipadic-mecab-2_7_0.tar.gz
-$ tar -xzf ipadic-mecab-2_7_0.tar.gz
+```shell
+# Specify an appropriate release tag to VERSION (e.g., v0.4.0).
+$ wget https://github.com/daac-tools/vibrato/releases/download/VERSION/ipadic-mecab-2_7_0.tar.xz
+$ tar -xzf ipadic-mecab-2_7_0.tar.xz
```
Comment (vbkaisetsu) [inline README.md:54]:
```suggestion
$ tar xf ipadic-mecab-2_7_0.tar.xz
```

**Review by vbkaisetsu — COMMENTED** [review]
It would be better to revert all code blocks for consistency.

**Comment by kampersanda** [comment]
@vbkaisetsu The reason why using `shell` syntax is to highlight comments in code blocks. So, I moved comments outside and reverted them. https://github.com/daac-tools/vibrato/pull/112/commits/48aefebcdd8a70901bd435d296569faabd77a4db

**Review by vbkaisetsu — APPROVED** [review]

**Pull request merged**